docs/applications/messaging/install-mastodon-on-ubuntu-1604/index.md

```diff
@@ -0,0 +1,460 @@
+
+1.  Pre-compile Mastodon's assets:
+
+        docker-compose run --rm web bundle exec rake assets:precompile
+
+    This command will take a while to finish.
+
+1.  Start Mastodon:
+
+        docker-compose up -d
+
+### Create your Mastodon User
+
+1.  Visit your domain in a web browser:
+
+    ![Mastodon Sign Up Screen](mastodon-sign-up-screen.png "Mastodon Sign Up Screen")
+
+1.  Enter a new username, email address, and password to create a new user on your Mastodon instance. When communicating with users of other Mastodon servers, your full username is `@your_username@example.com`.
+
+1.  Mastodon will attempt to send you a confirmation email. Check your email and click the link inside to confirm your registration.
+
+    If you did not set up email notifications, you can manually confirm the new user by running the `confirm_email` task on the Docker container:
+
+        docker-compose run web bundle exec rake mastodon:confirm_email USER_EMAIL=your_email_address
+
+1.  Run the `make_admin` task on your Docker container to make this new user an admin for the Mastodon instance:
+
+        docker-compose run web bundle exec rake mastodon:make_admin USERNAME=your_mastodon_instance_user
+
+##  Troubleshooting
+
+If your Mastodon site doesn't appear in your browser, try reviewing the logs generated by Docker for more information. To see these errors:
+
+1.  Shut down your containers:
+
+        cd mastodon
+        docker-compose down
+
+1.  Run Docker Compose in an attached state so that you can view the logs generated by each container:
+
+        docker-compose up
+
+1.  To shut down Docker Compose and return the command prompt again, enter `CTRL-C`.
+
+If your Mastodon site appears, but some specific site functions are not working, use the tools available in Mastodon's administrative settings, found through `example.com/admin/settings/edit`. Specifically, the Sidekiq dashboard can show the status of jobs issued by Mastodon:
+
+![Sidekiq Dashboard](sidekiq-dashboard.png "Sidekiq Dashboard")
+
+## Maintenance
+
+To update your Mastodon version, review the instructions from the Mastodon GitHub's [Docker Guide](https://github.com/tootsuite/documentation/blob/master/Running-Mastodon/Docker-Guide.md#updating). You should also review the [Mastodon release notes](https://github.com/tootsuite/mastodon/releases).
+
+### Renew your Let's Encrypt Certificate
+
+1.  Open your Crontab in your editor:
+
+        sudo crontab -e
+
+1.  Add a line which will invoke Certbot at 11PM every day. Replace `example.com` with your domain:
+
+        0 23 * * *   certbot certonly -n --webroot -w /usr/share/nginx/html -d example.com --deploy-hook='docker exec mastodon_nginx_1 nginx -s reload'
+
+1.  You can test your new job with the `--dry-run` option:
+
+        sudo bash -c "certbot certonly -n --webroot -w /usr/share/nginx/html -d example.com --deploy-hook='docker exec mastodon_nginx_1 nginx -s reload'"
```

Comment:
Request for the copy editor: I accidentally didn't include the `--dry-run` option here. Please update the line to be:

```
sudo bash -c "certbot certonly -n --webroot -w /usr/share/nginx/html -d example.com --deploy-hook='docker exec mastodon_nginx_1 nginx -s reload' --dry-run"
```